cli/dcoscli/marathon/main.py

```diff
@@ -872,3 +877,19 @@ def _cli_config_schema():
         pkg_resources.resource_string(
             'dcos',
             'data/config-schema/marathon.json').decode('utf-8'))
+
+
+class MarathonPodSubcommand(object):
+
+    def __init__(self, resource_reader, marathon_client):
+        self._resource_reader = resource_reader
+        self._marathon_client = marathon_client
```

Comment:
Created this class for the `pod` subcommand to make its dependencies explicit, and swappable in unit tests.